airbyte-webapp/src/components/base/Tooltip/InfoTooltip.module.scss

```diff
@@ -4,16 +4,17 @@
 
 $line-height: 17px;
 $icon-size: 14px;
+$icon-padding: 5px;
 $icon-top: math.div($line-height - $icon-size, 2);
 
 .container {
   display: inline-block;
-  margin-left: variables.$spacing-sm;
   color: colors.$grey;
-  width: $icon-size;
+  width: calc($icon-size + $icon-padding);
+  padding-left: $icon-padding;
 }
 
 .icon {
-  position: absolute;
+  position: relative;
   top: $icon-top;
```

Comment:
Yeah I made sure to verify that the other places using this component looked correct, and they seem to look right to me.

I made these styling changes in this component to correct the info icon positioning in Safari and Firefox, since KC pointed out above that the positioning was off in those browsers.

For example, this is what the streams table in the replication view looks like in Safari when the `.icon` selector has `position: absolute`:
<img width="1269" alt="Screen Shot 2022-09-16 at 2 45 00 PM" src="https://user-images.githubusercontent.com/22731524/190798225-62d843d1-3c5d-42af-9c9b-7dccbb7b2f9a.png">

With `position: relative`, the info icon positioning looks the same in Safari as it does in Firefox and Chrome:
<img width="1266" alt="Screen Shot 2022-09-16 at 2 46 04 PM" src="https://user-images.githubusercontent.com/22731524/190798722-e63097c8-bca9-4f7a-8fb6-1b702eab61a4.png">